storage/spring-cloud-azure-starter-storage-blob/storage-blob-sample/src/main/java/com/azure/spring/sample/storage/resource/StorageApplication.java

```diff
@@ -3,16 +3,51 @@
 
 package com.azure.spring.sample.storage.resource;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.core.io.WritableResource;
+
+import java.io.OutputStream;
 
 /**
  * @author Warren Zhu
  */
 @SpringBootApplication
-public class StorageApplication {
+public class StorageApplication implements CommandLineRunner {
```

Comment:
Why `CommandLineRunner` and web application at the same time?

Reply:
I want to use this CommandLineRunner to initialize the application, create some data in the blob containers so that users don't need to create some data in order to test AzureStorageBlobProtocolResolver.